README.md

```diff
@@ -8,23 +8,16 @@ English/[中文](docs/whatis.md)
 Sometimes we need to test performance of a deployed Fabric network with ease. There are many excellent projects out there, i.e. Hyperledger Caliper. However, we sometimes just need a tiny, handy tool, like `tape`.
 
 ## What is it
-
-This is a very simple traffic generator:
-- it does not use any SDK
-- it does not attempt to deploy Fabric
-- it does not rely on connection profile
-- it does not discover nodes, chaincodes, or policies
-- it does not monitor resource utilization
-
-It is used to perform super simple performance test:
-- it directly establishes number of gRPC connections
-- it sends signed proposals to peers via number of gRPC clients
-- it assembles endorsed responses into envelopes
-- it sends envelopes to orderer
-- it observes transaction commitment
```

Comment:
i thin we could leave the structure of this part unchanged - only change wording as you see fit